app/models/evaluator_submission_assignment.rb

```diff
@@ -22,25 +22,47 @@ class EvaluatorSubmissionAssignment < ApplicationRecord
     assigned: 0,
     unassigned: 1,
     recused: 2,
+    recused_unassigned: 3
+  }
+
+  def evaluation_status
+    if assigned?
+      if evaluation.nil?
+        :not_started
+      elsif evaluation.completed_at.nil?
+        :in_progress
+      else
+        :completed
+      end
+    else
+      status.to_sym
+    end
+  end
+
+  ORDER_VALUES = {
+    recused: 0,
+    unassigned: 1,
+    recused_unassigned: 2,
     not_started: 3,
     in_progress: 4,
-    completed: 5,
-    recused_unassigned: 6
-  }
+    completed: 5
+  }.freeze
+
+  def ordering_priority
+    case evaluation_status
+    when :recused then ORDER_VALUES[:recused]
+    when :unassigned then ORDER_VALUES[:unassigned]
+    when :recused_unassigned then ORDER_VALUES[:recused_unassigned]
+    when :not_started then ORDER_VALUES[:not_started]
+    when :in_progress then ORDER_VALUES[:in_progress]
+    when :completed then ORDER_VALUES[:completed]
+    end
```

Comment:
this can be reduced to one line?
```suggestion
    ORDER_VALUES[evaluation_status]
```